docker/Dockerfile

```diff
@@ -1,25 +1,27 @@
-FROM alpine:3.5
+FROM ubuntu:16.04
 
-RUN mkdir /opt
 WORKDIR /opt
 
 RUN NPROC=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || 1) && \
-    echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
-    apk update && \
-    apk upgrade && \
-    apk add git cmake wget make libc-dev gcc g++ bzip2-dev boost-dev zlib-dev expat-dev lua5.1-dev libtbb@testing libtbb-dev@testing && \
-    \
-    echo "Building libstxxl" && \
-    cd /opt && \
-    git clone --depth 1 --branch 1.4.1 https://github.com/stxxl/stxxl.git && \
-    cd stxxl && \
-    mkdir build && \
-    cd build && \
-    cmake -DCMAKE_BUILD_TYPE=Release .. && \
-    make -j${NPROC} && \
-    make install
+    apt-get update -y && \
+    apt-get install -y \
+        build-essential \
+        git \
+        cmake \
+        pkg-config \
+        libbz2-dev \
+        libstxxl-dev \
+        libstxxl1v5 \
+        libxml2-dev \
+        libzip-dev \
+        libboost-all-dev \
+        lua5.2 \
+        liblua5.2-dev \
+        libtbb-dev \
+        libjemalloc-dev
```

Comment:
`libtbb.so` is the only library we dynamically link to everything else gets statically linked into the osrm binaries and libs